TrkReco/src/MergeHelices_module.cc

```diff
@@ -141,25 +151,28 @@ namespace mu2e {
     unsigned nh1, nh2, nover;
     countHits(evt,h1,h2, nh1, nh2, nover);
     unsigned minh = std::min(nh1, nh2);
+    float chih1xy(0),chih1zphi(0),chih2xy(0),chih2zphi(0);
+    unsigned deltanh = 5; 
```

Comment:
This number needs to be fhicl-configurable.  It should also be a class member.  Please  add an appropriate atom to the Config struct, the set the value using that in the constructor.